### ISSUE TITLE
So where is `build` to run in produce?

### ISSUE THREAD

**Issue by jpluimers** [hintjens/mkbook#1]
Am I missing something or is `build` (mentioned in the README as `./build`) not yet part of the repository?

**Comment by hintjens** [comment]
The template for a new project is in example/, so you have example/produce/build

I need to fix this, I think... so mkbook is itself a template for a new project.

**Comment by hintjens** [comment]
See if it makes more sense now. I've not really had time to document this all properly.

**Comment by jpluimers** [comment]
Yup. Much clearer. I'll try to make some time to test this on a Mac. 
I will let you know if it worked out.

Just read you will be running out of earth-time, so don't worry about it too much: there are things way more important.
